generated/nifgen/nifgen.proto

```diff
@@ -70,7 +70,7 @@ service NiFgen {
   rpc CreateWaveformComplexF64(CreateWaveformComplexF64Request) returns (CreateWaveformComplexF64Response);
   rpc CreateWaveformF64(CreateWaveformF64Request) returns (CreateWaveformF64Response);
   rpc CreateWaveformFromFileF64(CreateWaveformFromFileF64Request) returns (CreateWaveformFromFileF64Response);
-  rpc CreateWaveformFromFileHws(CreateWaveformFromFileHwsRequest) returns (CreateWaveformFromFileHwsResponse);
+  rpc CreateWaveformFromFileHWS(CreateWaveformFromFileHWSRequest) returns (CreateWaveformFromFileHWSResponse);
```

Comment:
Note: I will revert these changes before adding reviewers

Reply:
Palash's PR for generated files went in just now. If you pull in latest, these files should disappear.

Reply:
@doshirohan , Yeah. I did not intend to send these changes for review. I kept this because Akshata was pulling this branch in Ubuntu VM to build and test there. I will remove the changes after she has pulled my branch.

I tried to remove reviewers till that but the github page was weird and did not allow me to remove

Reply:
DOne testing the new changes on Linux. And I have reverted those extra generated file changes. Hence resolving comment